topics/transcriptomics/tutorials/ref-based/tutorial.md

```diff
@@ -265,7 +265,7 @@ We should trim the reads to get rid of bases that were sequenced with high uncer
 
 > <hands-on-title>Trimming FASTQs</hands-on-title>
 >
-> 1. {% tool [Cutadapt](toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/4.8+galaxy1) %} with the following parameters to trim low quality sequences:
+> 1. {% tool [Cutadapt](toolshed.g2.bx.psu.edu/repos/lparsons/cutadapt/cutadapt/4.9+galaxy1) %} with the following parameters to trim low quality sequences:
```

Comment:
could the relevant workflows be updated as well if it's not too much bother?

Reply:
Done for the ref-based ones.
The QC one is so outdated and trivial at the same time that it would be easier to just recreate it from scratch, but I don't have time now anymore.

Reply:
sure, that's fine!